server/core/runtime/plan_step_runner_test.go

```diff
@@ -295,7 +295,7 @@ func TestRun_CreatesWorkspace(t *testing.T) {
 			// output of `workspace show` to be a different name.
 			When(terraform.RunCommandWithVersion(
 				ctx,
-				prjCtx, "/path", []string{"workspace", "show"}, map[string]string(nil), tfVersion, "workspace")).ThenReturn("diffworkspace\n", nil)
```

Comment:
Nope, it was intentional because of the change from checking string equivalence to substring here: https://github.com/lyft/atlantis/pull/275/files#diff-e17053a450204ccfcb0f37e78c0942ad10e685a2a91478e1738aa7ca9b7e9368R134